src/main/java/com/google/codeu/servlets/CalendarOAuthServlet.java

```diff
@@ -0,0 +1,39 @@
+package com.google.codeu.servlets;
+
+import com.google.api.client.auth.oauth2.AuthorizationCodeFlow;
+import com.google.api.client.auth.oauth2.AuthorizationCodeRequestUrl;
+import com.google.api.client.extensions.appengine.auth.oauth2.AbstractAppEngineAuthorizationCodeServlet;
+import com.google.api.client.googleapis.auth.oauth2.GoogleAuthorizationCodeFlow;
+import com.google.api.client.http.GenericUrl;
+import com.google.appengine.api.users.UserService;
+import com.google.appengine.api.users.UserServiceFactory;
+
+import java.io.IOException;
+import java.io.PrintWriter;
+
+import javax.servlet.ServletException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+public class CalendarOAuthServlet extends AbstractAppEngineAuthorizationCodeServlet {
+
+  @Override
+  protected void doGet(HttpServletRequest request, HttpServletResponse response)
+          throws IOException {
+    GoogleAuthorizationCodeFlow flow = Utils.newFlow();
```

Comment:
A high level comment here would be useful for teammates to understand what's going on. Nothing too detailed, but a little more explicit than what the code says.